### PR TITLE
Add VNC hash cracking (dictionary based)

### PR DESCRIPTION
**Pull request by realsdx**
This PR adds password hash cracking for VNC protocol. 
Using the top 10000 passwords from SecLists this tries to crack VNC password hash.

This PR also adds new parameter `password_hash` in auth logger.

### CODE REVIEW

File: heralding/capabilities/vnc.py [hunk 1]
```diff
@@ -55,8 +56,15 @@ class Vnc(HandlerBase):
         writer.write(challenge)
 
         client_response = await reader.read(1024)
-        session.add_auth_attempt('des_challenge', response=binascii.hexlify(client_response).decode(),
-                                 challenge=binascii.hexlify(challenge).decode())
-
         writer.write(AUTH_FAILED)
+
+        # try to decrypt the hash
+        dkey = decrypt_hash(challenge, client_response)
```
Comment (johnnykv) [inline heralding/capabilities/vnc.py]: Would be more correct to name the method crack_hash.

Comment (johnnykv) [inline heralding/capabilities/vnc.py]: What is dkey an abbreviation for?

File: heralding/misc/wordlist.py [hunk 2]
```diff
@@ -0,0 +1,10004 @@
+# form SecLists : https://github.com/danielmiessler/SecLists/blob/master/Passwords/xato-net-10-million-passwords-10000.txt
+
+plist = [
```
Comment (johnnykv) [inline heralding/misc/wordlist.py]: The wordlist should be included as a text file and loaded on startup

File: heralding/capabilities/vnc.py (hunk 1 above)
Comment (realsdx) [reply heralding/capabilities/vnc.py]: decrypted key.

File: heralding/misc/wordlist.py (hunk 2 above)
Comment (realsdx) [reply heralding/misc/wordlist.py]: okay.

**Comment by realsdx** [comment]
You mean rename 'decrypted'?

**Comment by johnnykv** [comment]
libs.decrypt.vnc

**Pull request closed without merging**